INSTRUCTION.md

```diff
@@ -0,0 +1,50 @@
+# INSTRUCTIONS.md
+
+## Django-Todolist: A Simple Todo List Web Application
+
+Django-Todolist is a minimal web application that combines essential features like user accounts, APIs, and a basic interactive UI. It uses CSS (Skeleton), JavaScript, and jQuery for styling and interactivity.
+
+### Prerequisites
+- Docker and Docker Compose installed on your system.
+
+### Running the Application with Docker Compose
+Follow these steps to run the Django-Todolist application:
+
+#### Step 1: Clone the Repository
+1. Clone the repository to your local machine:
+   ```bash
+   git clone https://github.com/MykhailoIvchenko/devops_todolist_docker_core_task_3_docker_compose.git
+   cd devops_todolist_docker_core_task_3_docker_compose
+   ```
+
+#### Step 2: Start the Application
+1. Use Docker Compose to build and start the containers:
+   ```bash
+   docker-compose up --build
+   ```
+2. This command will:
+   - Start the MySQL database service.
+   - Run database migrations automatically.
+   - Launch the Django-Todolist application.
+
+3. Once started, access the application in your web browser at:
+   [http://localhost:8000](http://localhost:8000).
```

Comment:
The instructions mention accessing the application at `http://localhost:8000`, but the Dockerfile exposes port 8080. Ensure that the exposed port in the Dockerfile and the port mentioned in the instructions are consistent.